modin/pandas/test/test_groupby.py

```diff
@@ -278,14 +286,14 @@ def __call__(self, df):
         ["col5", "col4"],
         ["col4", "col5"],
         ["col5", "col4", "col1"],
-        ["col1", pd.Series([1, 5, 7, 8])],  # 15
-        [pd.Series([1, 5, 7, 8])],
+        ["col1", pandas.Series([1, 5, 7, 8])],  # 15
+        [pandas.Series([1, 5, 7, 8])],
         [
-            pd.Series([1, 5, 7, 8]),
-            pd.Series([1, 5, 7, 8]),
-            pd.Series([1, 5, 7, 8]),
-            pd.Series([1, 5, 7, 8]),
-            pd.Series([1, 5, 7, 8]),
```

Comment:
Why changed? We process groupby differently depending on whether `by` is a modin or pandas object, so it's important to test cases of modin's Series. Please restore.

Reply:
The `pd.Series` constructor itself defaults to Pandas. The `pytestmark` warnings filter at the top of this file doesn't work for such calls in pytest case parameter definitions.

I used a separate variable for these series instead and made them modin series again. I left a comment explaining my rationale.